github/githubapp.go

```diff
@@ -65,7 +87,11 @@ func GenerateInstallationToken(ctx context.Context, client *http.Client, server
 	}
 	req.Header.Add("Accept", "application/vnd.github.v3+json")
 
-	jwtToken, err := generateJWTtoken(app.ClientId, app.PrivateKey)
+	if app.parsedRSAKey == nil {
+		return "", fmt.Errorf("misconfigured github_app: rsa key not configured. Call app.Validate() method first")
+	}
```

Comment:
Do we really need to be that paranoid? We shouldn't be in this function in the first place if we didn't get a private key as the GitHubApp did not validate.

Also the error message is funky. As a user, how am I supposed to call `app.Validate()` ?!?

Reply:
Removed...